apps/cowswap-frontend/src/modules/hooksStore/dapps/AirdropHookApp/styled/Input.tsx

```diff
@@ -0,0 +1,21 @@
+import { UI } from '@cowprotocol/ui'
+
+import styled from 'styled-components/macro'
+
+export const Input = styled.input`
```

Comment:
We usually create one file `styled.tsx` and put all styled elements there instead of creating a file per element. I would appreciate if you put all of those elements into one file